src/relay.js

```diff
@@ -288,12 +292,12 @@ export function sequelizeConnection({
         // In case of `OVER()` is not available, we need to get the full count from a second query.
         const options = await Promise.resolve(before({
           where: argsToWhere(args)
-        }));
+        }, args, context, info));
 
         if (target.count) {
-          fullCount = await target.count(source, options);
+          fullCount = await target.count(options);
         } else {
-          fullCount = await target.manyFromSource.count(source, options);
+          fullCount = await target.manyFromSource.count(options);
```

Comment:
Yes, that's the one I'm talking about. My case is a `target: Model`, and so in this case, I don't think we should pass the `source` argument. I can try to make my changes work with previous tests, and add some for `target: Model`.